src/pages/graphql/schema/checkout/queries/get-payment-config.md

```diff
@@ -268,6 +309,31 @@ Attribute | Data Type | Description
 
 See [Payment options](https://experienceleague.adobe.com/docs/commerce-merchant-services/payment-services/payments-checkout/payments-options.html) for more information.
 
+### `GooglePayConfig` attributes
+
+The `GooglePayConfig` payment method configuration contains the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`code` | String | The payment method code as defined in the payment gateway
+`is_visible` | Boolean | Indicates whether the payment method is shown
+`payment_intent` | String | Defines the payment intent. The possible values are `Authorize` or `Capture`
```

Comment:
```suggestion
`payment_intent` | String | Defines the payment intent. The possible values are `Authorize` or `Capture`.
```

Reply:
I had to make a lot of compromises along the way. No periods at the end of sentences or sentence fragments. This is consistent throughout the GraphQL reference docs